generators/app/templates/terra-consumer-package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "<%=projectName%>",
+  "main": "lib/<%=jsxFileName%>.js",
+  "version": "0.0.0",
+  "description": "{insert description}",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-consumer.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "Consumer",
+    "HealtheLife",
+    "<%=projectName%>",
+    "<%=jsxFileName%>",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-consumer/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-consumer#readme",
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "terra-base": "^1.0.0",
+    "terra-mixins": "^1.0.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "terra-base": "^1.0.0",
+    "terra-mixins": "^1.0.0"
+  },
+  "scripts": {
+    "compile": "npm run compile:clean && npm run compile:build",
+    "compile:clean": "$(cd ..; npm bin)/rimraf lib",
+    "compile:build": "$(cd ..; npm bin)/babel src --out-dir lib --copy-files",
+    "lint": "npm run lint:js && npm run lint:scss",
+    "lint:js": "$(cd ..; npm bin)/eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
+    "lint:scss": "$(cd ..; npm bin)/stylelint src/**/*.scss",
+    "props-table": "$(cd ..; npm bin)/props-table ./src/<%=jsxFileName%>.jsx --out-dir ./docs/props-table",
+    "test": "npm run test:spec && npm run test:nightwatch-default",
+    "test:spec": "$(cd ..; npm bin)/jest --config ../../jestconfig.json",
+    "test:all": "npm run test:nightwatch-default && npm run test:nightwatch-chrome && npm run test:nightwatch-firefox && npm run test:nightwatch-safari",
+    "test:nightwatch-default": "WEBPACK_CONFIG_PATH=../../../../terra-consumer-site/webpack.config SPECTRE_TEST_SUITE=<%= projectName %> node ../../node_modules/terra-toolkit/lib/scripts/nightwatch.js",
```

Comment:
These can be updated to:

`"test:nightwatch-default": " SPECTRE_TEST_SUITE=<%= projectName %> node ../../node_modules/terra-toolkit/lib/scripts/nightwatch.js",`